timer/src/main/java/com/toggl/timer/startedit/domain/ProjectTagChipSelector.kt

```diff
@@ -15,11 +15,11 @@ fun projectTagChipSelector(
     tags: Map<Long, Tag>
 ): List<ChipViewModel> = sequence {
 
-    if (editableTimeEntry.projectId != null) {
-        val project = projects[editableTimeEntry.projectId] ?: throw ProjectDoesNotExistException()
-        yield(ChipViewModel.Project(project))
-    } else {
-        yield(ChipViewModel.AddProject(addProjectLabel))
+    editableTimeEntry.projectId.let { projectId ->
+        if (projectId != null) {
+            val project = projects[projectId] ?: throw ProjectDoesNotExistException()
+            yield(ChipViewModel.Project(project))
+        } else yield(ChipViewModel.AddProject(addProjectLabel))
     }
```

Comment:
```suggestion
    editableTimeEntry.projectId?.let { projectId ->
        val project = projects[projectId] ?: throw ProjectDoesNotExistException()
        yield(ChipViewModel.Project(project))
    } ?: yield(ChipViewModel.AddProject(addProjectLabel))
```

The elvis operator looks alright for this case. Doesn't provide much of a readability improvement though.